---
bool released_for_manual_movement